prog/vnet/subnet_nexus.rb

```diff
@@ -37,7 +37,8 @@ def self.assemble(project_id, name: nil, location_id: Location::HETZNER_FSN1_ID,
       end
       firewall.associate_with_private_subnet(ps, apply_firewalls: false)
 
-      Strand.create(prog: "Vnet::SubnetNexus", label: "wait") { _1.id = ubid.to_uuid }
+      label = (location.provider == "aws") ? "create_vpc" : "wait"
+      Strand.create(prog: "Vnet::SubnetNexus", label:) { _1.id = ubid.to_uuid }
```

Comment:
Instead of two distinct entry points in this prog....would it make sense to have it figure things out on its own when it runs from a single entry point? Or is that perhaps too wasteful? It's unusual to have two entry points/startups for a prog is what I'm saying.